pkg/backup/actions/csi/pvc_action.go

```diff
@@ -64,14 +63,6 @@ func (p *pvcBackupItemAction) AppliesTo() (velero.ResourceSelector, error) {
 }
 
 func (p *pvcBackupItemAction) validateBackup(backup velerov1api.Backup) (valid bool) {
-	// Do nothing if volume snapshots have not been requested in this backup
```

Comment:
This means all the BIA plugins should not check `backup.Spec.SnapshotVolumes`, we may need to add this to release notes so as to advice the 3rd plugins to remove this check